src/moveit_python/planning_scene_interface.py

```diff
@@ -193,23 +204,21 @@ def addMesh(self, name, pose, filename, wait=True):
     def attachMesh(self, name, pose, filename, link_name, touch_links=None,
                    detach_posture=None, weight=0.0, wait=True):
         o = self.makeMesh(name, pose, filename)
+        r = self.makeMesh(name, pose, filename)
+        r.operation = r.REMOVE
```

Comment:
If you attach an object, it should automatically be [removed from the world](https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_core/planning_scene/src/planning_scene.cpp#L1457).
So this code is redundant.

Reply:
https://github.com/mikeferguson/moveit_python/commit/41b685ce7b9c8e516d20d9e3229d0fd8217bea80